docs/FAQ.md

```diff
@@ -18,10 +162,9 @@ intermediary loaders with a common Linux-based loader that provides a rich
 security processing framework. One role that TrenchBoot does not fulfill is
```

Comment:
Starting with the sentence "One role...", this section needs some rework and clarification. We were not exactly sure what that should look like because we did not quite understand what was trying to be conveyed here.

Ross

Reply:
This paragraph has been reworked in https://github.com/TrenchBoot/TrenchBoot.github.io/pull/45/commits/d2c5d71d0c6f66d06b5d3ae94754a8b5b700c3f5#diff-0bb42a85d04c8b7c9b659307979fd43e8f6e084283a300bfff959adfcbddfd17R122-R138